diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/createlinks mezzanine_patched_e-smith-apache-1.2.0/createlinks
--- e-smith-apache-1.2.0/createlinks	2006-03-14 09:47:18.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.2.0/createlinks	2007-01-19 07:49:25.000000000 -0700
@@ -162,3 +162,15 @@
 service_link_enhanced("httpd-e-smith", "S86", "7");
 service_link_enhanced("httpd-e-smith", "K15", "6");
 service_link_enhanced("httpd-e-smith", "K15", "0");
+
+# Set up generic logfile timestamp renaming/symlinking
+
+foreach (qw(
+    /var/log/httpd/error_log
+    /var/log/httpd/access_log
+    ))
+{
+    safe_touch "root/etc/e-smith/events/logrotate/logfiles2timestamp/$_";
+    safe_touch "root/etc/e-smith/events/post-install/logfiles2timestamp/$_";
+    safe_touch "root/etc/e-smith/events/post-upgrade/logfiles2timestamp/$_";
+}
