diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00	2007-02-13 13:28:13.000000000 -0500
@@ -0,0 +1,13 @@
+{
+    sub load_modules {
+        my @list = @_;
+        my $out = '';
+        foreach (@list)
+        {
+            next unless -f "/usr/lib/httpd/modules/mod_${_}.so";
+            $out .= "LoadModule ${_}_module modules/mod_${_}.so\n";
+        }
+        return $out;
+   }
+   '';
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule10 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule10
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule10	2004-11-09 15:36:38.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule10	2007-02-13 13:28:52.000000000 -0500
@@ -1,2 +1,3 @@
-LoadModule env_module         modules/mod_env.so
-LoadModule log_config_module  modules/mod_log_config.so
+{
+    $OUT .= load_modules(qw(env log_config));
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule20 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule20
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule20	2003-01-07 10:00:49.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule20	1969-12-31 19:00:00.000000000 -0500
@@ -1 +0,0 @@
-#LoadModule mime_magic_module  modules/mod_mime_magic.so
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule30 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule30
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule30	2004-11-09 15:36:38.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule30	2007-02-13 13:31:29.000000000 -0500
@@ -1,11 +1,16 @@
-LoadModule mime_module        modules/mod_mime.so
-LoadModule negotiation_module modules/mod_negotiation.so
-LoadModule status_module      modules/mod_status.so
-LoadModule info_module        modules/mod_info.so
-LoadModule include_module     modules/mod_include.so
-LoadModule autoindex_module   modules/mod_autoindex.so
-LoadModule dir_module         modules/mod_dir.so
-LoadModule cgi_module         modules/mod_cgi.so
-LoadModule asis_module        modules/mod_asis.so
-LoadModule imap_module        modules/mod_imap.so
-LoadModule actions_module     modules/mod_actions.so
+{
+    $OUT .= load_modules(qw(
+	    mime
+	    negotiation
+	    status
+	    info
+	    include
+	    autoindex
+	    dir
+	    cgi
+	    asis
+	    imap
+	    imagemap
+	    actions
+	));
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule40 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule40
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule40	2003-01-07 10:00:49.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule40	1969-12-31 19:00:00.000000000 -0500
@@ -1 +0,0 @@
-#LoadModule speling_module     modules/mod_speling.so
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule50 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule50
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule50	2006-03-14 11:47:18.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule50	2007-02-13 16:36:09.000000000 -0500
@@ -1,7 +1,13 @@
-LoadModule proxy_module       modules/mod_proxy.so
-LoadModule proxy_http_module  modules/mod_proxy_http.so
-LoadModule alias_module       modules/mod_alias.so
-LoadModule rewrite_module     modules/mod_rewrite.so
-LoadModule access_module      modules/mod_access.so
-LoadModule auth_module        modules/mod_auth.so
-LoadModule auth_anon_module   modules/mod_auth_anon.so
+{
+    $OUT .= load_modules(qw(
+		proxy
+		proxy_http
+		alias
+		rewrite
+		access
+		authz_host
+		authz_user
+		auth
+		auth_anon
+	    ));
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule60 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule60
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule60	2004-11-09 15:36:38.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule60	2007-02-13 13:33:44.000000000 -0500
@@ -1 +1,3 @@
-LoadModule auth_digest_module modules/mod_auth_digest.so
+{
+    $OUT .= load_modules(qw(auth_digest));
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule70 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule70
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule70	2003-01-07 10:00:49.000000000 -0500
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule70	2007-02-13 13:34:25.000000000 -0500
@@ -1,6 +1,8 @@
-LoadModule expires_module     modules/mod_expires.so
-LoadModule headers_module     modules/mod_headers.so
-LoadModule usertrack_module   modules/mod_usertrack.so
-#LoadModule example_module     modules/mod_example.so
-#LoadModule unique_id_module   modules/mod_unique_id.so
-LoadModule setenvif_module    modules/mod_setenvif.so
+{
+    $OUT .= load_modules(qw(
+		expires
+		headers
+		usertrack
+		setenvif
+	    ));
+}
