diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/createlinks mezzanine_patched_e-smith-apache-1.1.2/createlinks
--- e-smith-apache-1.1.2/createlinks   2005-07-19 16:37:32.783564712 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/createlinks 2005-07-19 16:34:58.980170208 -0600
@@ -46,6 +46,33 @@
 safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
 
 #--------------------------------------------------
+# actions for host-create event
+#--------------------------------------------------
+
+$event = "host-create";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
+# actions for host-delete event
+#--------------------------------------------------
+
+$event = "host-delete";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
+# actions for host-modify event
+#--------------------------------------------------
+
+$event = "host-modify";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
 # actions for ibay-create event
 #--------------------------------------------------
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup    1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup  2005-07-19 16:34:58.982169904 -0600
@@ -0,0 +1,9 @@
+{
+    use esmith::DomainsDB;
+    # Convert the passed hash for the domain object back into an object.
+    $domain = bless \%domain, 'esmith::DB::db::Record';
+
+    # Make scalars from some of the properties of the domain
+    $virtualHost = $domain->key;
+    $OUT = "";
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName       2005-07-19 16:37:32.664463800 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName     2005-07-19 16:34:58.984169600 -0600
@@ -1 +1 @@
-    ServerName www.{$virtualHost}
+    ServerName {$virtualHost}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias      2005-07-19 16:37:32.665464648 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias    2005-07-19 16:34:59.118149232 -0600
@@ -1 +1,8 @@
-    ServerAlias { "$virtualHost  $SystemName.$virtualHost" }
+    ServerAlias {
+
+    use esmith::HostsDB;
+    my $hdb = esmith::HostsDB->open_ro();
+    my @hosts = grep { $_->prop('HostType') eq 'Self' } $hdb->get_hosts_by_domain($virtualHost);
+    $OUT = join " ", map { $_->key } @hosts;
+}
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName        2003-01-07 08:00:49.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName      2005-07-19 16:34:58.987169144 -0600
@@ -1 +1 @@
-    ServerName www.{$virtualHost}
+    ServerName {$virtualHost}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias       2003-07-02 13:47:24.000000000 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias     2005-07-19 16:34:59.120148928 -0600
@@ -1 +1,8 @@
-    ServerAlias { "$virtualHost  $SystemName.$virtualHost" }
+    ServerAlias {
+
+    use esmith::HostsDB;
+    my $hdb = esmith::HostsDB->open_ro();
+    my @hosts = grep { $_->prop('HostType') eq 'Self' } $hdb->get_hosts_by_domain($virtualHost);
+    $OUT = join " ", map { $_->key } @hosts;
+}
+
