diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/db/configuration/defaults/modSSL/CipherSuite mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/db/configuration/defaults/modSSL/CipherSuite
--- e-smith-apache-1.1.2/root/etc/e-smith/db/configuration/defaults/modSSL/CipherSuite	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/db/configuration/defaults/modSSL/CipherSuite	2005-12-15 11:07:36.948624475 +1100
@@ -0,0 +1 @@
+ALL:!ADH:RC4+RSA:+HIGH:+MEDIUM:-LOW:+SSLv2:-EXP
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35SSL10SSLCipherSuite mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35SSL10SSLCipherSuite
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35SSL10SSLCipherSuite	2005-12-15 11:07:57.479052412 +1100
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35SSL10SSLCipherSuite	2005-12-15 11:06:48.524049807 +1100
@@ -1 +1,3 @@
-SSLCipherSuite ALL:!ADH:RC4+RSA:+HIGH:+MEDIUM:-LOW:+SSLv2:-EXP
+{
+    $OUT = "SSLCipherSuite $modSSL{CipherSuite}";
+}
