diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/createlinks mezzanine_patched_e-smith-apache-1.2.0/createlinks
--- e-smith-apache-1.2.0/createlinks	2007-09-11 14:01:57.000000000 -0600
+++ mezzanine_patched_e-smith-apache-1.2.0/createlinks	2007-09-11 14:00:13.000000000 -0600
@@ -16,7 +16,7 @@
 $event = "bootstrap-console-save";
 
 templates2events("/etc/httpd/conf/httpd.conf", $event);
-templates2events("/etc/logrotate.d/apache", $event);
+templates2events("/etc/logrotate.d/httpd", $event);
 
 #--------------------------------------------------
 # actions for domain-create event
