diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/createlinks mezzanine_patched_e-smith-apache-1.1.2/createlinks
--- e-smith-apache-1.1.2/createlinks	2005-07-19 00:16:37.081978376 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/createlinks	2005-07-19 00:16:18.780760584 -0600
@@ -46,6 +46,33 @@
 safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
 
 #--------------------------------------------------
+# actions for host-create event
+#--------------------------------------------------
+
+$event = "host-create";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
+# actions for host-delete event
+#--------------------------------------------------
+
+$event = "host-delete";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
+# actions for host-modify event
+#--------------------------------------------------
+
+$event = "host-modify";
+
+templates2events("/etc/httpd/conf/httpd.conf", $event);
+safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/httpd-e-smith");
+
+#--------------------------------------------------
 # actions for ibay-create event
 #--------------------------------------------------
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/00Setup	2005-07-18 23:33:12.520876240 -0600
@@ -0,0 +1,9 @@
+{
+    use esmith::DomainsDB;
+    # Convert the passed hash for the domain object back into an object.
+    $domain = bless \%domain, 'esmith::DB::db::Record';
+
+    # Make scalars from some of the properties of the domain
+    $virtualHost = $domain->key;
+    $OUT = "";
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName	2005-07-19 00:16:36.942999504 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/02ServerName	2005-07-18 23:21:13.506816768 -0600
@@ -1 +1 @@
-    ServerName www.{$virtualHost}
+    ServerName {$virtualHost}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias	2005-07-19 00:16:36.946998896 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/ProxyPassVirtualHosts/03ServerAlias	2005-07-19 00:07:38.517114392 -0600
@@ -1 +1,11 @@
-    ServerAlias { "$virtualHost  $SystemName.$virtualHost" }
+    ServerAlias { 
+
+    use esmith::HostsDB;
+    my $hdb = esmith::HostsDB->open_ro();
+
+    my @localdomains = map { $_->key => 1 } $hdb->get_all_by_prop( HostType => 'Self' );
+    my @thisdomain = grep /\.$virtualHost$/, @localdomains;
+
+    $OUT = join " ", @thisdomain;
+}
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName	2003-01-07 08:00:49.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/02ServerName	2005-07-18 23:25:55.011809008 -0600
@@ -1 +1 @@
-    ServerName www.{$virtualHost}
+    ServerName {$virtualHost}
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias	2003-07-02 13:47:24.000000000 -0600
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias	2005-07-19 00:07:39.847912080 -0600
@@ -1 +1,11 @@
-    ServerAlias { "$virtualHost  $SystemName.$virtualHost" }
+    ServerAlias { 
+
+    use esmith::HostsDB;
+    my $hdb = esmith::HostsDB->open_ro();
+
+    my @localdomains = map { $_->key => 1 } $hdb->get_all_by_prop( HostType => 'Self' );
+    my @thisdomain = grep /\.$virtualHost$/, @localdomains;
+
+    $OUT = join " ", @thisdomain;
+}
+
