diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/75AddType05.xsl mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/75AddType05.xsl
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/75AddType05.xsl	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/75AddType05.xsl	2005-12-14 15:23:52.765034315 +1100
@@ -0,0 +1 @@
+AddType application/xml .xsl
