diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias	2006-03-14 09:47:18.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/VirtualHosts/03ServerAlias	2007-02-23 09:03:41.000000000 -0700
@@ -1,8 +1,13 @@
     ServerAlias {
 
     use esmith::HostsDB;
+    use esmith::DomainsDB;
     my $hdb = esmith::HostsDB->open_ro();
-    my @hosts = grep { $_->prop('HostType') eq 'Self' } $hdb->get_hosts_by_domain($virtualHost);
+    my %domains = map { $_ => 1 } esmith::DomainsDB->open_ro->domains;
+    my @hosts = grep {
+            my $h = $_;
+            ($h->prop('HostType') eq 'Self') && !$domains{$h->key}
+        } $hdb->get_hosts_by_domain($virtualHost);
     $OUT = join " ", map { $_->key } @hosts;
 }
 
