diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80mod_perl mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80mod_perl
--- e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80mod_perl	2006-01-31 11:01:45.384224276 +1100
+++ mezzanine_patched_e-smith-apache-1.1.2/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80mod_perl	2006-01-31 10:54:52.038141849 +1100
@@ -1,8 +1,7 @@
 {
-    # vim: ft=perl:
-    my $status = $modPerl{status} || 'enabled';
-    if ( $status eq "enabled" )
-    {
-	$OUT = "LoadModule perl_module        modules/mod_perl.so";
-    }
+    my $status = $modPerl{status}; 
+
+    return "# modPerl disabled" unless ($status and $status eq "enabled");
+    
+    return "LoadModule perl_module	modules/mod_perl.so";
 }
