diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00 mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00	2007-05-18 21:29:24.000000000 -0600
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule00	2007-05-18 21:28:54.000000000 -0600
@@ -4,7 +4,7 @@
         my $out = '';
         foreach (@list)
         {
-            next unless -f "/usr/lib/httpd/modules/mod_${_}.so";
+            next unless -f "/usr/lib/httpd/modules/mod_${_}.so" || "/usr/lib64/httpd/modules/mod_${_}.so";
             $out .= "LoadModule ${_}_module modules/mod_${_}.so\n";
         }
         return $out;
diff -Nur -x '*.orig' -x '*.rej' e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35ExternalPwAuth mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35ExternalPwAuth
--- e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35ExternalPwAuth	2004-11-09 13:36:39.000000000 -0700
+++ mezzanine_patched_e-smith-apache-1.2.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/35ExternalPwAuth	2007-05-18 21:05:31.000000000 -0600
@@ -1,3 +1,3 @@
-AddExternalAuth pwauth /usr/lib/httpd/modules/pwauth
+AddExternalAuth pwauth { -f "/usr/lib64/httpd/modules/pwauth" ? "/usr/lib64/httpd/modules/pwauth" : "/usr/lib/httpd/modules/pwauth" }
 SetExternalAuthMethod pwauth pipe
 
